hermes-frontend/src/main/java/pl/allegro/tech/hermes/frontend/publishing/HttpMetaHeaders.java

```diff
@@ -0,0 +1,6 @@
+package pl.allegro.tech.hermes.frontend.publishing;
+
+public interface HttpMetaHeaders {
```

Comment:
So we need either mapping between the HTTP header and Hermes header or use the same name for both.